presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -203,6 +203,12 @@ Accessors
     For GeometryCollection types, returns the sum of the areas of the individual
     geometries.
 
+.. function:: ST_Area(SphericalGeography) -> double
+
+    Returns the spherical Area of a geography, using a spherical model for Earth
```

Comment:
Perhaps, clarify the units: `Returns the area of a polygon or multi-polygon in meters using a spherical model for Earth.`